src/supermarket/app/controllers/api/v1/quality_metrics_controller.rb

```diff
@@ -63,21 +62,6 @@ def collaborators_evaluation
     head 200
   end
 
-  #
-  # POST /api/v1/cookbook-versions/license_evaluation
-  #
-  # License evaluation is handled by Foodcritic now, so this endpoint
-  # is deprecation and will no longer accept License metric results.
-  # If the +CookbookVersion+ does not exist, render a 404 not_found.
-  #
-  # Will return a 410 Gone because this resource has been intentionally removed
-  #
-  def license_evaluation
-    response = { message: "Endpoint deprecated. License metric results are now produced by Foodcritic." }
```

Comment:
We should keep this, but update the message from Foodcritic -> Cookstyle since it does the evaluation as well

Reply:
This can actually go as well since it was never documented.